### ISSUE TITLE
FEATURE - Add ability to define positive and negative axis for joystick input separately

### ISSUE THREAD

**Issue by howels** [CurryKitten/CurryKitten-Sim#15]
Currently the input selector picks an axis but you cannot specify which direction is + and which is -, they will always follow the scheme defined by the controller. The result is that the defaults which LiftOff uses are somehow inverted for pitch and throttle versus what CurryKitten-Sim uses.
Would be great to push the joystick in the + direction when selecting the + axis and in the - direction when selecting the - axis. Not a deal breaker but would be useful.

**Comment by CurryKitten** [comment]
I think this looks like a direction I need to go.  Although every actual radio I've plugged into was perfect and happy, there's a large section of users with weird and wonderful which do things like: put yaw on a potentiometer which can't be moved.  I also just tried plugging in a regular USB joystick, a PS4 one (which I figure people who don't currently fly might like to try on) and every single axis was wrong.

Plan here is to keep the simple common channel maps along with modes 1-4 but have an extra button which lets you customise the axis.  At which point the code will ask the user to push the throttle to it's max etc in order to decide which axis go where and which direction is positive
